textproc/txt2html/Portfile

```diff
@@ -1,12 +1,12 @@
 PortSystem 1.0
+PortGroup	perl5 1.0
+PortGroup	github 1.0
 
-name		txt2html
-version		1.35
+github.setup	resurrecting-open-source-projects txt2html 3.0 v
```

Comment:
please add github.tarball_from" and set to appropriate value: archive or releases. Likely will require updating the "checksums".